tests/core/usage_statistics/test_usage_stats_schema.py

```diff
@@ -128,14 +130,31 @@ def test_data_context_get_batch_list_message():
             # record itself
             jsonschema.validate(
                 message,
-                usage_statistics_record_schema,
+                anonymized_usage_statistics_record_schema,
             )
             jsonschema.validate(
                 message["event_payload"],
                 anonymized_batch_request_schema,
             )
 
 
+def test_checkpoint_run_message():
```

Comment:
Should there also be a similar test for `data_context.run_checkpoint()`?

Reply:
@anthonyburdi Probably not necessary, because the payload is empty.  However, it should be added to the event list -- I will double check.